cvat/apps/consensus/signals.py

```diff
@@ -17,6 +17,5 @@
 def __save_task__initialize_consensus_settings(instance: Task, created, **kwargs):
     # Initializes default quality settings for the task
     # this is done in a signal to decouple this component from the engine app
-
-    if created and instance.consensus_replicas:
+    if created and instance.consensus_replicas and not kwargs.get("raw"):
```

Comment:
It would be safer and more readable to add `raw: bool` to the function argument list.